docs/community/com-ingest/com-ingest-manage.md

```diff
@@ -36,21 +39,23 @@ Ingesters are used to:
 
 ## CSV import method
 
-Refer to these reference files for information on how to set up and import data using a CSV file.
+Learn how to create source files containing table structure and data, then to import them on the CLI.
```

Comment:
I've seen a lot of CLI name changes coming in and am worried. This is not the CLI. these are the consumers/ingester. fbsql is the only CLI we have, and saying this everywhere is going to cause a lot of confusion

Reply:
>fbsql is the only CLI we have

That's news to me. 

I thought you said the SQL CLI referred to in /community/com-config/com-config-sql-cli-enable.md is still a thing?

Or is that just for HTTP stuff?

Reply:
Those are the same thing. The community page is now out of date and can probably be deleted

Reply:
I'll confirm this next week. For now I've rehashed the explanation and standardised the anchor text.